randovania/games/samus_returns/logic_database/Area 5 - Exterior.txt

```diff
@@ -99,17 +95,13 @@ Extra - asset_id: collision_camera_002
   * Extra - actor_type: doorpowerpower
   > Pickup (Super Missile Tank Top)
       Screw Attack and Space Jump
-  > Top of Tower Right Side
+  > Next to Teleporter
       Any of the following:
-          High Jump Boots or Space Jump or Damage Boost (Intermediate) or Shoot Beam Burst or Simple IBJ
-          Spiderspark (Beginner) and Can Spiderspark
-          Knowledge (Beginner) and Lay Power Bomb
+          High Jump Boots or Space Jump or Damage Boost (Intermediate) or Destroy Blob Throwers/Steel Orbs or Simple IBJ
           Phase Drift and Movement (Intermediate)
-  > Next to Teleporter
+  > Right Side Middle Ledge
       Any of the following:
-          High Jump Boots or Space Jump or Damage Boost (Intermediate) or Shoot Beam Burst or Simple IBJ
-          Spiderspark (Beginner) and Can Spiderspark
-          Knowledge (Beginner) and Lay Power Bomb
+          High Jump Boots or Space Jump or Damage Boost (Intermediate) or Destroy Blob Throwers/Steel Orbs
```

Comment:
`or Simple IBJ`?
Guess it's to get over the Blob Throwers thing?

Reply:
Yes it is. I forgot to re-add it